docs/streams/developer-guide/config-streams.html

```diff
@@ -1111,28 +1111,31 @@ <h4><a class="toc-backref" href="#id18">Default Values</a><a class="headerlink"
           <td>Consumer</td>
```

Comment:
Just saw the following above:
```
For detailed descriptions
        of these configs, see <a class="reference external" href="http://kafka.apache.org/0100/documentation.html#producerconfigs">Producer Configs</a>
        and <a class="reference external" href="http://kafka.apache.org/0100/documentation.html#newconsumerconfigs">Consumer Configs</a>.</p>
```
Still links to 0.10.0... Can we fix it?

Reply:
We also set `max.poll.records` to 1000.

Reply:
Version fixed. For `max.poll.records` it was added in the previous commit